vite.base.config.mts

```diff
@@ -7,6 +7,7 @@ export default defineConfig({
       namedExport: false,
     }),
   ],
+  esbuild: { target: "es2022" },
```

Comment:
This is necessary for `vitest` to run properly. The issue and workaround I found can be found [here](https://github.com/vitest-dev/vitest/issues/4183#issuecomment-1780959907).